### PR TITLE
examples for aws/eks

### PR DESCRIPTION
**Pull request by cofyc**
### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: examples/aws/tidb-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v4.0.2
```
Comment (DanielZhangQD) [inline examples/aws/tidb-cluster.yaml:6]: Suggest adding the following parameters:
configUpdateStrategy: RollingUpdate
nodeSelector for each component

File: examples/aws/tidb-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  timezone: UTC
+  pvReclaimPolicy: Delete
```
Comment (DanielZhangQD) [inline examples/aws/tidb-cluster.yaml:8]:
```suggestion
  pvReclaimPolicy: Retain
```

File: examples/aws/tidb-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  pd:
+    baseImage: pingcap/pd
+    replicas: 1
```
Comment (DanielZhangQD) [inline examples/aws/tidb-cluster.yaml:11]:
```suggestion
    replicas: 3
```

File: examples/aws/tidb-cluster.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+apiVersion: pingcap.com/v1alpha1
```
Comment (DanielZhangQD) [inline examples/aws/tidb-cluster.yaml:1]: This is used in the official EKS deployment doc, so we have to provide a yaml file that can be used in production env, please include the necessary configurations in the https://github.com/pingcap/tidb-operator/blob/master/deploy/aws/manifests/db.yaml.example

File: examples/aws/tidb-monitor.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    service:
+      portName: http-grafana
+      type: LoadBalancer
+    username: admin
+    version: 6.1.6
+  imagePullPolicy: IfNotPresent
+  initializer:
+    baseImage: pingcap/tidb-monitor-initializer
+    imagePullPolicy: IfNotPresent
+    version: v4.0.0
```
Comment (DanielZhangQD) [inline examples/aws/tidb-monitor.yaml:33]:
```suggestion
    version: v4.0.2
```

File: examples/aws/tidb-cluster.yaml (hunk 1 above)
Comment (cofyc) [reply examples/aws/tidb-cluster.yaml:6]: nodeSelector is not necessary as we already use `dedicated` taint

Comment (cofyc) [reply examples/aws/tidb-cluster.yaml:6]: `configUpdateStrategy: RollingUpdate` is added

Comment (DanielZhangQD) [reply examples/aws/tidb-cluster.yaml:6]: If not adding `nodeSelector`, we have to add taint to the node group for operator and monitor which we do not do that now and also have to make it clear how to add toleration for operator and monitor.

File: examples/aws/tidb-cluster.yaml (hunk 4 above)
Comment (cofyc) [reply examples/aws/tidb-cluster.yaml:1]: done

File: examples/aws/tidb-cluster.yaml (hunk 1 above)
Comment (cofyc) [reply examples/aws/tidb-cluster.yaml:6]: in the old aws terraform scripts, we already use `taints` to reverse node groups for our components (pd, tikv, etc.)

what I did is to remove redundant node selector configuration

other components, tidb-operator are deployed in control node group (in my new docs, I named it `admin`)

File: examples/aws/tidb-monitor.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    service:
+      portName: http-grafana
+      type: LoadBalancer
+    username: admin
+    version: 6.1.6
+  imagePullPolicy: IfNotPresent
+  initializer:
+    baseImage: pingcap/tidb-monitor-initializer
+    imagePullPolicy: IfNotPresent
+    version: v4.0.2
```
Comment (DanielZhangQD) [inline examples/aws/tidb-monitor.yaml:33]:
```suggestion
    version: v4.0.3
```

**Pull request merged**